msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/perf/TestCasePerf.java

```diff
@@ -0,0 +1,168 @@
+                .startAuthorizationFromActivity(mActivity)
+                .withLoginHint(mLoginHint)
+                .withScopes(Arrays.asList(mScopes))
+                .withCallback(successfulInteractiveCallback(latch))
+                .withPrompt(Prompt.SELECT_ACCOUNT)
+                .build();
+
+
+        final InteractiveRequest interactiveRequest = new InteractiveRequest(
+                mApplication,
+                parameters,
+                new OnInteractionRequired() {
+                    @Override
+                    public void handleUserInteraction() {
+                        final String username = mLoginHint;
+                        final String password = LabConfig.getCurrentLabConfig().getLabUserPassword();
+
+                        final PromptHandlerParameters promptHandlerParameters = PromptHandlerParameters.builder()
+                                .prompt(PromptParameter.SELECT_ACCOUNT)
+                                .loginHint(mLoginHint)
+                                .sessionExpected(false)
+                                .consentPageExpected(false)
+                                .speedBumpExpected(false)
+                                .build();
+
+                        new AadPromptHandler(promptHandlerParameters)
+                                .handlePrompt(username, password);
+                    }
+                }
+        );
+
+        interactiveRequest.execute();
+        latch.await(TokenRequestTimeout.SHORT);
+
+        final IAccount account = getAccount();
+        CodeMarkerManager.getInstance().setEnableCodeMarker(true);
+        //Setting up scenario code. 100 -> Non Brokered, 200 -> Brokered
+        CodeMarkerManager.getInstance().setPrefixScenarioCode("100");
+
+        for(int i = 0; i < numberOfOccurrenceOfTest; i++) {
+            CodeMarkerManager.getInstance().clearMarkers();
+            final TokenRequestLatch silentLatch = new TokenRequestLatch(1);
+
+            final AcquireTokenSilentParameters silentParameters = new AcquireTokenSilentParameters.Builder()
+                    .forAccount(account)
+                    .fromAuthority(account.getAuthority())
+                    .forceRefresh(true)
+                    .withScopes(Arrays.asList(mScopes))
+                    .withCallback(successfulSilentCallback(silentLatch))
+                    .build();
+
+            mApplication.acquireTokenSilentAsync(silentParameters);
+            silentLatch.await(TokenRequestTimeout.SILENT);
+            try {
+                FileAppender fileAppender = new FileAppender(outputFilenamePrefix + i + ".txt", new SimpleTextFormatter());
+                fileAppender.append(CodeMarkerManager.getInstance().getFileContent());
+                CommonUtils.copyFileToFolderInSdCard(
+                        fileAppender.getLogFile(),
+                        "automation"
+                );
+            } catch (IOException e) {
+                e.printStackTrace();
```

Comment:
the test should probably fail at this point

Reply:
Done. Thanks